test/e2e/framework/framework.go

```diff
@@ -259,6 +259,80 @@ func (f *Framework) matchNginxConditions(name string, matcher func(cfg string) b
 	}
 }
 
+// GetNginxConfigMap gets ingress-nginx's nginx-configuration map
+func (f *Framework) GetNginxConfigMap() (*v1.ConfigMap, error) {
```

Comment:
this function does not seem to be used in anywhere else outside of this package, IMHO we should not export it

Reply:
Sounds good!

Reply:
I'm not sure I feel strongly about doing that since even if we don't namesapce this method to the framework struct, the methods are still usable in the tests. For example, see how `UpdateDeployment` is used here: https://github.com/kubernetes/ingress-nginx/blob/master/test/e2e/lua/dynamic_configuration.go#L89. 

The benefit of namespacing it to framework is that we wouldn't need to pass in a `KubeClientSet` like we need to when calling `UpdateDeployment`. Let me know what you think